tests/test-form-data-redirect.js

```diff
@@ -5,9 +5,10 @@ var path = require('path')
 var request = require('../')
 var fs = require('fs')
 var tape = require('tape')
+var destroyable = require('server-destroy')
 
 function runTest (t, options) {
-  var localFile = path.join(__dirname, 'unicycle.jpg')
+  var localFile = path.join(__dirname, 'googledoodle.jpg')
```

Comment:
?

Reply:
The previous image was bigger in size which made this test flaky on Node v10 because of previously discussed stream issue on redirects.